R/load_survey.r

```diff
@@ -38,14 +38,15 @@ load_survey <- function(files, ...) {
   ## first, get the common files
   for (type in main_types)
   {
-    main_file <- grep(paste0("_", type, "_common.*\\.csv$"), survey_files, value = TRUE)
+    main_file <- grep(paste0("_", type, "s?_common.*\\.csv$"), survey_files, value = TRUE)
```

Comment:
As far as I can tell, this change in the the regex doesn't have any effect so it may be better to revert it to keep this diff focused on the merge issue.

Reply:
It's to mitigate the filename change in [this update](https://zenodo.org/record/7326686#.Y334YLLP23I) on Zenodo (now called `2020_Zhang_China_contacts_common.csv`) and thinking it's a so easily made mistake that might be good to ignore contact vs. contacts and participant vs. particpants. Agree that it's out of place here though so I'll put it in a separate PR.

Reply:
#86